recipes/sanic/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sanic" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "f70fbd2483e372494d023a08259ffbeb757b3bce10bbf4a00b17a405de3ff322" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: >
+    set SANIC_NO_UVLOOP=true &&  # [win]
```

Comment:
ah right, sorry I forgot how Windows works. :)

Reply:
See my comment below, that gives me a headache.

Reply:
Somehow it's getting an extra space in there, reading the value as `'true '`. You could try using instead:
```yaml
  script:
    - set SANIC_NO_UVLOOP=true  # [win]
    - python setup.py install --single-version-externally-managed --record record.txt
```